test/browser_tests/spec_suites/molecules/multiselect.js

```diff
@@ -0,0 +1,208 @@
+'use strict';
+
+var Blog = require(
+    '../../page_objects/page_blog.js'
+  );
+
+describe( 'Multiselect', function() {
+  var page;
+
+  beforeEach( function() {
+    page = new Blog();
+    page.get();
+
+    page.searchFilterBtn.click();
+    browser.sleep( 1000 );
+  } );
+
+  describe( 'on page load', function() {
+    it( 'should create the multiselect', function() {
+      expect( page.multiSelect.isDisplayed() ).toBe( true );
+    } );
+
+    it( 'should not show the dropdown', function() {
+      expect( page.multiSelectFieldset.isDisplayed() ).toBe( false );
+    } );
+  } );
+
+  describe( 'when interacting with search input', function() {
+    it( 'should show the dropdown on click', function() {
+      page.multiSelectSearch.click();
+      browser.sleep( 1000 );
+
+      expect( page.multiSelectFieldset.isDisplayed() ).toBe( true );
+    } );
+
+    xit( 'should show the dropdown on focus', function() {
+      // Couldn't get this to tab over, not sure why
```

Comment:
Do you mind adding a note to that effect for people trying to fix these tests later?